### ISSUE TITLE
Postprocessing fails for a specific show - second word picked up as language by guessit

### ISSUE THREAD

**Issue by PinaColonic** [pymedusa/Medusa#4826]
**Describe the bug**
I have a new show called Making It but postprocessor fails and I cannot figure out why.

**To Reproduce**
Manually download Making It on RARBG - I went with 1080P WEB TBS release
Add show Making It premiered in 2018 from TVMAZE
Manual Post Process, and it fails logs below
**Expected behavior**
Successful Postprocessing

**Screenshots**
N/A

**Medusa (please complete the following information):**
 - OS: Linux Mint 19
 - Branch: master
 - Commit: f5973b2

**Logs:**
Logs in details below / need to expand
<details>

2018-08-01 07:24:25 WARNING  Thread_10 :: [f5973b2] /medusa/Making.It.S01E01.1080p.WEB.x264-TBS[rarbg]/making.it.s01e01.1080p.web.x264-tbs.mkv: Processing failed: This show isn't in your list, you need to add it before post-processing an episode
2018-08-01 07:24:25 WARNING  Thread_10 :: [f5973b2] Problem(s) during processing, failed for the following files/folders: 
2018-08-01 07:24:25 WARNING  Thread_10 :: [f5973b2] Processing failed for /medusa/Making.It.S01E01.1080p.WEB.x264-TBS[rarbg]/making.it.s01e01.1080p.web.x264-tbs.mkv: This show isn't in your list, you need to add it before post-processing an episode
</details>

**Additional context**
The show is called Making It - should be no scene exception name in advanced settings or anything, I don't understand why it is failing to postprocess when dozens of other shows work OK.

**Comment by p0psicles** [comment]
My guess is that guessit has problens parsing it. Can you go into the test folder. And run report_guessit release_name from command line?

**Comment by PinaColonic** [comment]
p0psicles:  Stupid question, what command line?  Linux?     Not sure how to get into a Medusa specific command line.

mint@mint /medusa/Making.It.S01E01.1080p.WEB.x264-TBS[rarbg] $ report_guessit release_name
report_guessit: command not found
mint@mint /medusa/Making.It.S01E01.1080p.WEB.x264-TBS[rarbg] $

I tried with the entire MKV file instead of release_name just in case that is what you wanted which also failed with that command not found.

**Comment by p0psicles** [comment]
No go into your medusa app folder. Go into the test folder. There run  `python report_guessit.py name.of.the.file.mkv`

**Comment by PinaColonic** [comment]
This look right?  I did tests folder as I didn't see a test.

mint@mint ~/Medusa-master/tests $ python report_guessit.py /report_guessit.py /media/download/sickie/Making.It.S01E01.1080p.WEB.x264-TBS[rarbg]/making.it.s01e01.1080p.web.x264-tbs.mkv
# guessit: 2.1.4  rebulk: 0.9.0
# show list: ['/medusa/Making.It.S01E01.1080p.WEB.x264-TBS[rarbg]/making.it.s01e01.1080p.web.x264-tbs.mkv']
? /report_guessit.py
  title: report guessit py
  mimetype: text/x-python
  type: episode
  parsing_time: 0.0865828990936
mint@mint ~/Medusa-master/tests $

**Comment by p0psicles** [comment]
No not a folder. Just pass the release name. But it between double quotes.

**Comment by PinaColonic** [comment]
p0psicles:  Sorry, I'm trying to work too so distracted and I screwed up the command above, I had an extra /report_guessit.py in there after the first one....facepalm.    

I appreciate your help.     I ran both with and without quotes + folder name, looks like it thinks the Language is "It" for some weird reason - like Italian maybe?

mint@mint ~/Medusa-master/tests $ python report_guessit.py /medusa/making.it.s01e01.1080p.web.x264-tbs.mkv
# guessit: 2.1.4  rebulk: 0.9.0
? /medusa/making.it.s01e01.1080p.web.x264-tbs.mkv
: title: making
  language: it
  season: 1
  episode: 1
  screen_size: 1080p
  format: WEB-DL
  video_codec: h264
  video_encoder: x264
  release_group: tbs
  container: mkv
  mimetype: video/x-matroska
  type: episode
  parsing_time: 0.104757070541
mint@mint ~/Medusa-master/tests $ 

and 

mint@mint ~/Medusa-master/tests $ python report_guessit.py "making.it.s01e01.1080p.web.x264-tbs.mkv"
# guessit: 2.1.4  rebulk: 0.9.0
? making.it.s01e01.1080p.web.x264-tbs.mkv
: title: making
  language: it
  season: 1
  episode: 1
  screen_size: 1080p
  format: WEB-DL
  video_codec: h264
  video_encoder: x264
  release_group: tbs
  container: mkv
  mimetype: video/x-matroska
  type: episode
  parsing_time: 0.0847918987274
mint@mint ~/Medusa-master/tests $

**Comment by p0psicles** [comment]
Yeah i suspected this. Will have to check later if this is fixed with guessit version 3.

**Comment by medariox** [comment]
You can add `Making` as scene exception in the meantime as workaround. It's a guess it issue.

**Comment by ErroneousBosch** [comment]
Seeing this same issue, scene exception works, but am also seeing that Medusa is not finding any episodes on searching unless the scene exception is in.

**Comment by sharkykh** [comment]
Will be fixed in version 0.2.10 of Medusa.

Result on with guessit 3:
```
$ python tests/report_guessit.py "making.it.s01e01.1080p.web.x264-tbs.mkv"
# guessit: 3.0.0  rebulk: 0.9.0
? making.it.s01e01.1080p.web.x264-tbs.mkv
: title: making it
  season: 1
  episode: 1
  screen_size: 1080p
  source: Web
  video_codec: H.264
  video_encoder: x264
  release_group: tbs
  container: mkv
  mimetype: video/x-matroska
  type: episode
  parsing_time: 0.0929999351501
```
